packages/client/src/generation/generateClient.ts

```diff
@@ -146,7 +146,7 @@ export async function generateClient({
   clientVersion,
   engineVersion,
   activeProvider,
-}: GenerateClientOptions): Promise<BuildClientResult | undefined> {
+}: GenerateClientOptions): Promise<{}> {
```

Comment:
Does it make sense to use just `Promise<void>` at this point?

Reply:
Could make sense indeed.
Let's get this merged today asap though.